client/src/components/chess-game/ChessGame.js

```diff
@@ -14,6 +14,15 @@ import './ChessGame.css';
  * and while running server or building server it exports function
 */
 const Chess = typeof ChessJs === 'function' ? ChessJs : ChessJs.Chess;
+const blackPlayerBoard = (squares, board) => {
+  const newSquares = Object.assign([], squares);
```

Comment:
You can use Array.from(squares) to create new array, which is more intuitive.

Reply:
Done.